auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.21-SNAPSHOT</version>
+    <version>1.0.21</version>
```

Comment:
This does not look right. We should have `*-SNAPSHOT` versions in our modules on `master`. The "snapshot" suffix is removed during release CI.